assets/js/book.js

```diff
@@ -0,0 +1,32 @@
+function Book(title, author, pages, read) {
+  this.id = localStorage.uniqueId;
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+
+  this.info = function() {
+    return "title: " + this.title + " author: " + this.author +
+           " pages: " + this.pages + " read: " + this.read;
+  }
+
+  this.toggleRead = function() {
+    this.read = !this.read;
+  }
+
+  this.render = function() {
```

Comment:
I am not very comfortable about how we implemented this. I would look into implementing this using `document.createElement` rather than hardcoding all our html elements like this. [This](https://coderwall.com/p/o9ws2g/why-you-should-always-append-dom-elements-using-documentfragments) and [this](https://stackoverflow.com/questions/2946656/advantages-of-createelement-over-innerhtml) might help to enlighten you on that.

Reply:
I'm not sure that it's the complete correct way to implement with using DOM functions. I know it's much efficient but our project doesn't need the efficiency much more than the code readability. I think our version is way better than on readability. I think that if we need to use DOM manipulation, we should prefer using javascript frameworks such as React or Angular. Anyway, we implemented your way to learning how to do as well but commented it.